src/components/Button.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const name = props.buttonName;// eslint-disable-line
```

Comment:
Let's not rule out the line and use proper destructuring to take the prop should look something like this ` const { buttonName } = props` just like you did on the display component.